lm_eval/datasets/quac/quac.py

```diff
@@ -39,8 +39,8 @@
 
 _HOMEPAGE = "https://quac.ai/"
 
-# TODO: Add the licence for the dataset here if you can find it
-_LICENSE = ""
+# License available at https://quac.ai/
+_LICENSE = "CC BY-SA 4.0"
```

Comment:
Oddly the datasheet: https://quac.ai/datasheet.pdf claims the license is MIT for the dataset, but I think you're right we should use the one from this page instead?

Reply:
There's a genuine and explicit disagreement between these two sources and I don't see a way to resolve it. CC-BY-SA 4.0 makes more sense insofar as it's a content license (as opposed to software like MIT) but I don't see a solution other than contacting the authors.

Reply:
I have contacted the authors to ask. Seeing as how a CC BY-SA 4.0 license is consistent with but stricter than a MIT license I recommend using that for now.